numba/datamodel/testing.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import print_function, absolute_import
```

Comment:
Hmm... I think it's more readable if this all goes into numba.tests.test_datamodel... Unless it's also used for CUDA testing?